R/zscored_annotation.R

```diff
@@ -5,7 +5,7 @@
 #' "Sample"  Unique SampleIDs used in your RNAseq dataset
 #' "FusionName" GeneA--GeneB ,or if fusion is intergenic then Gene1A/Gene2A--GeneB
 #'
-#' @param standardFusionCalls Annotates standardizes fusion calls from callers STARfusion| Arriba or QC filtered fusion
+#' @param standardFusioncalls Annotates standardizes fusion calls from callers STARfusion| Arriba or QC filtered fusion
```

Comment:
Good catch on this one. Guess it never was a problem till now as we did not have examples 😬

Reply:
yeah 🙈